nav2_lifecycle_manager/src/lifecycle_manager_client.cpp

```diff
@@ -26,10 +26,16 @@ namespace nav2_lifecycle_manager
 {
 using nav2_util::geometry_utils::orientationAroundZAxis;
 
-LifecycleManagerClient::LifecycleManagerClient(const std::string & name)
+LifecycleManagerClient::LifecycleManagerClient(
+  const std::string & name,
+  std::string namespace_)
 {
-  manage_service_name_ = name + std::string("/manage_nodes");
-  active_service_name_ = name + std::string("/is_active");
+  if (namespace_ != "") {
+    namespace_ += "/";
```

Comment:
The rclcpp node below has a constructor to take in the namespace, I'm not sure why you're adding it here. you could just do `node_ = std::make_shared<rclcpp::Node>(name + "_service_client", namespace);`

Reply:
Sure, I could, though I am not sure what's the best solution.
If we use the namespace to create the _service_client node, it will be under the same namespace as the manager it is controlling, and a different than the instance that created it.

For example, if in a process `process_containing_client_nav` started without a namespace, we have a manager client `client_nav` controlling a manager `lifecycle_manager_navigation` under the `lm_ns` namespace:
```
  nav2_lifecycle_manager::LifecycleManagerClient client_nav("lifecycle_manager_navigation", "lm_ns");
```
The _service_client node will be created under the manager namespace:

> $ ros2 node list
> /process_containing_client_nav
> /lm_ns/lifecycle_manager_localization
> /lm_ns/lifecycle_manager_localization_bond_client
> /lm_ns/lifecycle_manager_localization_service_client

Whereas if we don't use the manager namespace to create the _service_client node, it will be in the same namespace as the process that started it:

> $ ros2 node list
> /process_containing_client_nav
> /lifecycle_manager_localization_service_client
> /lm_ns/lifecycle_manager_localization
> /lm_ns/lifecycle_manager_localization_bond_client

I honestly don't know what's best

Reply:
I'd argue the first is more correct, because you need to namespace the `lifecycle_manager_localization` in order to run multiple instances of it, which I assume is the point of this PR.

Reply:
Ok, convinced